lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -5,7 +5,7 @@
 <%namespace name='static' file='/static_content.html'/>
 <%inherit file="/main.html" />
 
-<%block name="bodyclass">view-teams is-in-course course js</%block>
+<%block name="bodyclass">view-teams is-view-in-course course js</%block>
```

Comment:
Nit picking here, but I tend to prefer either:

* a SMACSS based stateful class name here, like you previously had (``.is-in-course``)
* or a BEMish structural class name (``.view-in-course``)

This combination of both seems a bit wordy and not as easy to follow. What do you think?